mongoexport/options.go

```diff
@@ -33,7 +33,7 @@ type OutputFormatOptions struct {
 	Type string `long:"type" value-name:"<type>" default:"json" default-mask:"-" description:"the output format, either json or csv"`
 
 	// Deprecated: allow legacy --csv option in place of --type=csv
-	CSVOutputType bool `long:"csv" default:"false" hidden:"true"`
```

Comment:
What's up with the `default:"false"` tags being removed?

Reply:
This was a design change in the go-flags library that we had to account for. It throws an error if you attempt to use a default tag on a boolean flag.

This is where the change was introduced: https://github.com/jessevdk/go-flags/commit/7f2ab82552ae1b12e070838918ac056507c0f114#diff-103d48dbdf9e16c6eb09b90668a9e984R279